pom.xml

```diff
@@ -36,9 +36,11 @@
 	</distributionManagement>
 
 	<properties>
+		<project.parent.version>${project.version}</project.parent.version>
 
 		<!-- Dependency versions -->
 		<spring-cloud-dependencies.version>2021.0.5</spring-cloud-dependencies.version>
+		<spring-cloud-gcp-dependencies.version>${project.parent.version}</spring-cloud-gcp-dependencies.version>
```

Comment:
This combination works in building either from root or submodules (spring-cloud-gcp-starters, spring-cloud-previews aggregator) but still looks kind of sketchy. Any better ideas?